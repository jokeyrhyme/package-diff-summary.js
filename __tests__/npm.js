/* @flow */
'use strict'

const npm = require('../lib/npm.js')

test('nameToMarkdown("does-not-exist")', () => {
  const name = 'does-not-exist-' + (Math.random() * 1e6)
  return npm.nameToMarkdown(name)
    .then((result) => {
      expect(result).toBe(name)
    })
})

test('nameToMarkdown("execa")', () => {
  return npm.nameToMarkdown('execa')
    .then((result) => expect(result).toMatchSnapshot())
})
