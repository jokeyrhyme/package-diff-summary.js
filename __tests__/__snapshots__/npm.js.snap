exports[`test nameToMarkdown("execa") 1`] = `"[execa](https://www.npmjs.com/package/execa)"`;
